torch/nn/modules/loss.py

```diff
@@ -290,6 +290,11 @@ class KLDivLoss(_Loss):
     .. _Kullback-Leibler divergence:
         https://en.wikipedia.org/wiki/Kullback-Leibler_divergence
 
+    .. note:: The default averaging means that the loss is actually **not** the
```

Comment:
Nit: you should mention `size_average` and tell the users how to get the correct KL div behavior.
Something like:

To get the real KL Divergence, use the ``size_average=False``, then divide the output of the module by the number of minibatches.

Example:
   loss = nn.KLDiv(size_average=False)
   out = loss(tensor1, tensor2) / tensor1.size(0)